packages/presentation/lib/screen/home/widgets/home_grid_view.dart

```diff
@@ -42,6 +43,10 @@ class _HomeGridViewState extends State<HomeGridView> {
 
   @override
   Widget build(BuildContext context) {
+    final size = MediaQuery.of(context).size;
+    final width = size.width;
+    final height = size.height;
+    int crossAxisCountSize = Responsive.isDesktop(context) ? 4 : 2;
```

Comment:
is 4 axis will be enougth for full desktop wide? mb better smth like this: 
  int crossAxisCountSize = wide/itemWide